app/workers/reactions/bust_homepage_cache_worker.rb

```diff
@@ -0,0 +1,21 @@
+module Reactions
+  class BustHomepageCacheWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
+
+    def perform(reaction_id, cache_buster = CacheBuster)
+      reaction = Reaction.find_by(id: reaction_id, reactable_type: "Article")
+      return unless reaction&.reactable
+
+      featured_articles_ids = Article.where(featured: true).order("hotness_score DESC").limit(3).pluck(:id)
+      return unless featured_articles_ids.include?(reaction.reactable_id)
+
+      reaction.reactable.touch
+      cache_buster.bust("/")
```

Comment:
BTW, is there a reason we have `cache_buster.bust("/")` twice in a row?

Reply:
It's something that has been done in the old job. I don't know the real reason why we're caching the same path twice, but I think we can remove the duplicate. What's your opinion @rhymes?

Reply:
I am not sure why it is there but I remember hearing something about making sure the cache is truly empty, thus the double post (as this is the homepage's cache, which is probably the most important). 

I think we should keep it in until we trace the why of this to its source and then we can decide if it's superfluous.

Reply:
@rhymes Okay.